Menu/Menu.js

```diff
@@ -1,10 +1,19 @@
 
 const toggleMenu = () => {
   // Toggle the "menu--open" class on your menu refence. 
+  menu.classList.toggle('menu--open')
 }
 
 // Start Here: Create a reference to the ".menu" class
-const menu;
+const menu = document.querySelector('.menu');
+console.log(menu);
+
+
 // create a reference to the ".menu-button" class
-const menuButton;
+const menuButton = document.querySelector('.menu-button');
+console.log(menuButton);
+
+
+
 // Using your menuButton reference, add a click handler that calls toggleMenu
+menuButton.addEventListener('click', e => toggleMenu())
```

Comment:
since toggleMenu isn't looking for or using any parameters, and it also does not use the THIS keyword, you can write this event listener more directly without the ES6 syntax

```
menuButton.addEventListener('click', toggleMenu);
```